### DISCUSSION TITLE
a little advice

### DISCUSSION THREAD

**Discussion by ewetdans** [IBAX-io/go-ibax#26] [Ideas] [answered]
![ibax33](https://user-images.githubusercontent.com/96885317/148484458-c550cc66-88aa-4eaa-bb2f-3ad8cf63b02b.jpg)

In the coin sending window, the back and send buttons look like a single button. It would be better to separate it with a line.
I wish you the best of luck in this beautiful project. Best regards

**Comment by HalloKyle** [comment, marked as answer]
That's a good one, We will fix that shortly!!! Thank you so much please leave your Twitter handle so we can reward you for the good advice.!

**Reply by ewetdans** [reply]
thanks.
my twitter: @danskursu

**Reply by ewetdans** [reply]
> That's a good one, We will fix that shortly!!! Thank you so much please leave your Twitter handle so we can reward you for the good advice.!
thanks.
my twitter: @danskursu

**Comment by ewetdans** [comment]
teşekkürler.
twitterım: @danskursu

**Comment by 1453oguz** [comment]
Ok my Twetter
@AloneKi24285567

18 Oca 2022 Sal 14:39 tarihinde ewetdans ***@***.***> şunu
yazdı:

**Comment by hrhrrcp** [comment]
Twitter Adress
@hrhrrcp
________________________________
Gönderen: 1453oguz ***@***.***>
Gönderildi: 19 Ocak 2022 Çarşamba 18:18
Kime: IBAX-io/go-ibax ***@***.***>
Bilgi: Subscribed ***@***.***>
Konu: Re: [IBAX-io/go-ibax] a little advice (Discussion #26)


Ok my Twetter
@AloneKi24285567

18 Oca 2022 Sal 14:39 tarihinde ewetdans ***@***.***> şunu
yazdı:

> That's a good one, We will fix that shortly!!! Thank you so much please
> leave your Twitter handle so we can reward you for the good advice.!
> thanks.
> my twitter: @danskursu
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/IBAX-io/go-ibax/discussions/26#discussioncomment-1989891>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXKFE5V3WOXM4FJHNSP5KFTUWVGQ3ANCNFSM5LN3N67A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/26#discussioncomment-1998658>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXL3UY6VEIQRGNJXKWH7PCDUW3I3PANCNFSM5LN3N67A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
